railties/lib/rails/generators/rails/app/templates/Gemfile

```diff
@@ -34,7 +34,7 @@ group :development, :test do
   gem 'byebug', platforms: [:mri, :mingw, :x64_mingw]
   <%- if depends_on_system_test? -%>
   # Adds support for Capybara system testing and selenium driver
-  gem 'capybara', '~> 2.13.0'
+  gem 'capybara', '~> 2.13'
```

Comment:
I did intentionally make it `2.13` because it had upstream changes I wanted. I didn't mean to make it so strict though.

One thing to keep in mind is that we don't really _test_ anything against Capybara on our CI so a bump could easily break something unexpected.